src/views/Login/Login.js

```diff
@@ -47,6 +47,19 @@ const SignIn = () => {
         event.preventDefault();
       };
 
+      const resetpassword = async () => {
+        toaster(t('Check your email to reset password'), 'success')
+        const response = await fetch(`${REACT_APP_BACKEND_URL}/user/resetpassword`,
+        {
+          method:'POST',
+          body:values.email,
```

Comment:
This can not work since it send empty email. And also make sure to send in headers `Accept-Language` just to support multiple language